src/sortStudents.ts

```diff
@@ -1,16 +1,56 @@
-
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
+
+export function sortStudents(students:Student[], sortBy: SortType,
+  order: SortOrder): Student[] {
```

Comment:
You should choose the strategy of how to write parameters inline or from the new line and use it for EVERY parameter
E-g: sortStudents(students:Student[], sortBy: SortType, order: SortOrder) or 
```suggestion
export function sortStudents(
  students: Student[],
  sortBy: SortType,
  order: SortOrder
): Student[] {
```